python-sdk/tests/benchmark/synthetic_ndjson_generator.py

```diff
@@ -0,0 +1,62 @@
+"""
+Function to generate ndjson files of various sizes
+"""
+import ndjson
+from faker import Faker
+
+fake = Faker()
+
+
+# Define function to generate fake data and store into a NDJSON file
+def generate_data(file_name, records):
+    customer = [
+        {
+            "id": n,
+            "name": fake.name(),
+            "address": fake.address(),
+            "email": str(fake.email()),
+            "phone": str(fake.phone_number()),
+        }
+        for n in range(0, records)
+    ]
+    # Write the data into the NDJSON file
+    with open(file_name, "w") as fp:
+        ndjson.dump(customer, fp)
+
+    print(f"File {file_name} has been created.")
+
+
+# create ten_kb ndjson file
+ten_kb = "fake_dataset/ndjson/ten_kb.ndjson"
```

Comment:
@sunank200 I think we can make this quicker for text-based data:
1. Create a file of 1MB or so
2. Make copy of the file generated
3. Then concrete it. `cat copy1.ndjson copy2.ndjson copy3.ndjson > newFile.ndjson`

Reply:
Well, there is an `id` which I wanted to keep unique.